pkgs/top-level/all-packages.nix

```diff
@@ -24395,7 +24395,9 @@ in
 
   oberon-risc-emu = callPackage ../misc/emulators/oberon-risc-emu { };
 
-  obs-studio = libsForQt5.callPackage ../applications/video/obs-studio { };
+  obs-studio = libsForQt5.callPackage ../applications/video/obs-studio {
+    ffmpeg = ffmpeg-full.override { srt = srt; };
```

Comment:
Did we already check if inherit srt works here?

Reply:
I tried it and it did not work. As I have absolutely no idea how to debug the error message i switched back to the override version.

```diff
diff --git a/pkgs/top-level/all-packages.nix b/pkgs/top-level/all-packages.nix
index 906e90a8249..6f9433dace7 100644
--- a/pkgs/top-level/all-packages.nix
+++ b/pkgs/top-level/all-packages.nix
@@ -24395,7 +24395,9 @@ in
 
   oberon-risc-emu = callPackage ../misc/emulators/oberon-risc-emu { };
 
-  obs-studio = libsForQt5.callPackage ../applications/video/obs-studio { };
+  obs-studio = libsForQt5.callPackage ../applications/video/obs-studio {
+    ffmpeg = ffmpeg-full { inherit srt; };
+  };
 
   obs-wlrobs = callPackage ../applications/video/obs-studio/wlrobs.nix { };
```

```
[nix-shell:~/Code/nixpkgs]$ nix-build -A obs-studio --show-trace
error: while evaluating the attribute 'buildInputs' of the derivation 'obs-studio-26.1.2' at /home/obs/Code/nixpkgs/pkgs/stdenv/generic/make-derivation.nix:192:11:
while evaluating 'getOutput' at /home/obs/Code/nixpkgs/lib/attrsets.nix:482:23, called from undefined position:
while evaluating anonymous function at /home/obs/Code/nixpkgs/pkgs/stdenv/generic/make-derivation.nix:143:17, called from undefined position:
attempt to call something which is not a function but a set, at /home/obs/Code/nixpkgs/pkgs/top-level/all-packages.nix:24399:14
```